flux-dsl/src/main/java/com/influxdb/query/dsl/functions/restriction/ColumnRestriction.java

```diff
@@ -189,12 +189,15 @@ public String toString() {
 
             String value;
             if (fieldValue instanceof String) {
-                value = "\"" + fieldValue + "\"";
+                value = "\"" + escapeDoubleQuotes((String) fieldValue) + "\"";
             } else {
                 value = FunctionsParameters.serializeValue(fieldValue);
             }
 
-            return "r[\"" + fieldName + "\"] " + operator + " " + value;
+            return "r[\"" + escapeDoubleQuotes(fieldName) + "\"] " + operator + " " + value;
         }
     }
+    private static String escapeDoubleQuotes(final String val) {
```

Comment:
Please add an empty line:

```suggestion
    
    private static String escapeDoubleQuotes(final String val) {
```